data/transactions/logic/opcodes_test.go

```diff
@@ -35,6 +35,12 @@ func TestOpSpecs(t *testing.T) {
 func TestOpcodesByVersion(t *testing.T) {
 	t.Parallel()
 
+	// To test the case where a newer version opcode is before an older version
```

Comment:
make this not Parallel() since it is modifying a global resource. I think the comment on OpcodesByVersion() may be out of date and my reading of that function right now, I don't see where it modifies the OpSpecs global? We could add to this test something that makes a copy of the global OpSpecs, runs OpcodesByVersion(), and checks that the global OpSpecs matches the original copy?
Then, at the end of this test, undo the swap and put OpSpecs back so that no other code sees any difference.

Reply:
>  I think the comment on OpcodesByVersion() may be out of date and my reading of that function right now

This is correct, I changed it but forgot to update the comment.

Reply:
Done.